src/main/java/frc/team2412/robot/Robot.java

```diff
@@ -99,7 +100,12 @@ public void robotInit() {
 	}
 
 	@Override
-	public void testInit() {}
+	public void testInit() {
+		CommandScheduler.getInstance()
+				.schedule(
+						new diagnosticSequentialCommand(
+								subsystems.intakeSubsystem, subsystems.launcherSubsystem));
```

Comment:
Did this go through spotless to standardize the whitespacing?
It would be good to align and remove extra white spaces to align the arguments.

Reply:
![image](https://github.com/robototes/Crescendo2024/assets/145405364/2def3b48-87a7-4b7f-a76c-5adf747a2a17)
I mean it looked like this, and this seems normal so i dont know